tool/tsh/tsh.go

```diff
@@ -460,6 +466,20 @@ func Run(args []string, opts ...cliOption) error {
 		return trace.Wrap(err)
 	}
 
+	// parameter must be yes, no, auto or only
+	switch cf.AddKeysToAgent {
+	case "yes":
+		break
```

Comment:
these don't need a break - they're not fallthrough as in C:
```go
switch cf.AddKeysToAgent {
	case "yes", "no", "auto", "only":
	default:
		return trace.BadParameter("valid values for parameter add-keys-to-agent are yes, no, auto or only, got %s", cf.AddKeysToAgent)
}
```

Reply:
and use `%q` to print the wrong value to catch whitespace